Assets/Scripts/Scriptable Objects/SlamBehaviour.cs

```diff
@@ -0,0 +1,71 @@
+using UnityEngine;
+using UnityEngine.AI;
+
+[CreateAssetMenu(fileName = "Slam Behaviour", menuName = "Behaviours/Slam")]
+public class SlamBehaviour : AIBehaviour
+{
+    public float range = 10f;
+    public float radius = 2f;
+    public int damage = 1;
+    public float cooldown = 5f;
+    private float timeSinceLastJump = 0;
+    private bool jumping = false;
+    private float slamSearchRange = 0.5f;
+    private Vector3 startPosition;
+    private Vector3 endPosition;
+    private float distance = 1f;
+    private float jumpTime = 0f;
+    public override void Act(ref Knowledge knowledge)
+    {
+        if (jumping)
+        {
+            jumpTime += (Time.deltaTime / distance) * knowledge.Info.speed;
+            knowledge.Agent.isStopped = true;
+            knowledge.AITransform.position = WanderAI.Parabola(startPosition, endPosition, Mathf.Sqrt(distance / range) * range, jumpTime);
+            if (jumpTime >= 1f)
+            {
+                foreach (Collider hitCollider in Physics.OverlapSphere(knowledge.AITransform.position, radius))
```

Comment:
A more performant option could be to restrict the `OverlapSphere` to exclude everything except the player layer and use `OverlapSphereNonAlloc` instead. However, this is a tiny optimisation and does not need to be implemented.